src/client/scripts/createRoom.js

```diff
@@ -119,21 +117,12 @@ Template.createRoom.events({
         var customCategory = Session.get("categoryToAdd");
 
         // prevents dupicates
```

Comment:
This has become a NS comment, should probably just kill.

Also, you changed behavior on this, it used to short circuit the method and return early if there was a duplicate tag, but this will actually accept it (though it will still display the error message) and let you create the room/add the new category